ckanext/extractor/config.py

```diff
@@ -61,18 +64,18 @@ def get(setting):
 
 
 # Adapted from ckanext-archiver
-def load_config(ini_path):
-    """
-    Load CKAN configuration.
-    """
-    ini_path = os.path.abspath(ini_path)
-    logging.config.fileConfig(ini_path, disable_existing_loggers=False)
-    conf = paste.deploy.appconfig('config:' + ini_path)
-    load_environment(conf.global_conf, conf.local_conf)
-    _register_translator()
-
-
-# Adapted from ckanext-archiver
+# def load_config(ini_path):
```

Comment:
@tino097 Just a reminder to clean this up and remove if not required anymore

Reply:
i miss this one, will make new PR if needed

Reply:
No rush,  just include it in the next PR for the extractor.